portfolio/src/main/webapp/script.js

```diff
@@ -112,3 +112,12 @@ function showPostForm() {
 function setNicknameForm() {
     window.open("/login");
 }
+
+function sendFollowedTags() {
+  const tagElem = document.getElementById('followtags');
+
+  const params = new URLSearchParams();
+  params.append('tags', tagElem.value);
```

Comment:
I think this will cause an error. tagElem has multiple values so you can loop through the options to access each value and check whether it was selected. (ie. for (var i = 0; i < tagElem.options.length; i++) {...}
And then you can access each option as tagElem.options[i]

Reply:
Probably the easiest way is to directly access each option with their id's since you have id's for each option in the index file. Then you can check which ones were selected if any.

Reply:
I tested it out and from what I can tell it would appear that value of the tagElem is the value of the option selected so that way I wouldn't need to iterate through the list but that also means that the user can only pick one at a time and has to re submit if they want to follow multiple tags.

Reply:
ok. we can use that for now and try to improve it later